pkg/reconciler/v1alpha1/broker/broker.go

```diff
@@ -395,45 +386,42 @@ func newChannel(b *v1alpha1.Broker, l map[string]string) *v1alpha1.Channel {
 	}
 }
 
-// TriggerChannelLabels are all the labels placed on the Trigger Channel for the given Broker. This
+// TriggerChannelLabels are all the labels placed on the Trigger Channel for the given brokerName. This
 // should only be used by Broker and Trigger code.
-func TriggerChannelLabels(b *v1alpha1.Broker) map[string]string {
+func TriggerChannelLabels(brokerName string) map[string]string {
```

Comment:
Why the change?

Reply:
I'm using this and the other label functions in the UTs. 
I didn't want to create a broker every time just to use the name. I can roll this back if you want

Reply:
I like it better this way, I have been making similar changes.

Reply:
I agree that it makes tests a little harder, but I prefer this way as it lets us actually use the type system well. For the most part, there is only one Broker object floating around but a very large number of strings.

Reply:
OK, will do the change in another PR